statsmodels/regression/tests/test_lme.py

```diff
@@ -583,7 +620,10 @@ def test_vcomp_formula(self):
         model1 = MixedLM(endog, exog, groups, exog_re=exog_re, exog_vc=exog_vc)
         result1 = model1.fit()
 
-        df = pd.DataFrame(exog[:, 1:], columns=["x1", ])
+        df = pd.DataFrame(
+            exog[:, 1:], columns=[
+                "x1",
+            ])
```

Comment:
Breaking the columns list into three lines is pretty ugly.  Standard thing to do here would be to keep the original but remove the trailing `, ` inside the `columns` list.